CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -68,6 +70,28 @@ static function CreateRecoverPersonnel (out array<X2DataTemplate> Templates)
 	Templates.AddItem(Activity);
 }
 
+static function CreatePreparePersonnel (out array<X2DataTemplate> Templates)
+{
+	local X2ActivityTemplate_CovertAction Activity;
+	local X2CovertActionTemplate CovertAction;
+	
+	`CREATE_X2TEMPLATE(class'X2ActivityTemplate_CovertAction', Activity, name("Activity_PreparePersonnel"));
```

Comment:
Why `name("")`? Why not just `''`?

Reply:
Because that's how it's done in the whole DefaultActivities file. Not sure why, but I went with established practice

Reply:
It's a bad practice, dunno why we did it like that